src/main/views/viewManager.ts

```diff
@@ -422,7 +419,7 @@ export class ViewManager {
 
     setLoadingScreenBounds = () => {
         if (this.loadingScreen) {
-            this.loadingScreen.setBounds(getWindowBoundaries(this.mainWindow));
+            this.loadingScreen.setBounds(getWindowBoundaries(MainWindow.get()!));
```

Comment:
you could add a cons for the main window and avoid unwrapping here adding the check in the if statement